bin/setup.sh

```diff
@@ -1,90 +1,93 @@
 #!/bin/bash
 ROOT=$(cd "$(dirname "${BASH_SOURCE[0]}")" && cd .. && pwd)
+cd ${ROOT}
 
-TITLE="WP-Docker-Construct"
-ADMIN_EMAIL="engenharia@log.pt"
-REPOSITORY="git@github.com:log-oscon/WP-Construct.git"
+TITLE=${PWD##*/}
 
 echo "-----------------------------"
 echo "${TITLE}"
 echo "-----------------------------"
 
-read -p "Write your domain: " site_url
-URL="${site_url// /-}.local";
+read -p "Write your domain | default: localhost |: " site_url
 
-read -p "is Multisite (y/n)? default n: " is_multisite
-case "$is_multisite" in 
-  y|Y ) MULTISITE=1;;
-  n|N ) MULTISITE=0;;
-  * ) MULTISITE=0;;
-esac
+if [ -z "${site_url}" ]; then
+  URL='localhost.local'
+else 
+  URL=${site_url// /-};
+fi
 
-## CHECKOUT PROJECT ##
-if [ ! -z "$REPOSITORY" ] && [ ! -d "./wordpress/.git" ]; then
-	echo "No local repository found. Downloading boilerplate / project from GIT..."
-  cd "$ROOT/wordpress"
-	git clone $REPOSITORY .
-  cd "$ROOT"
+if ! echo "${URL}" | grep '^[a-zA-Z0-9]*\.[a-zA-Z]*$' >/dev/null; then
+	URL="${URL}.local";
 fi
 
-## PROJECT DEPENDENCIES ##
-echo "Searching for project build script..."
-if [ -f "./wordpress/.scripts/build.sh" ]; then
-	cd "$ROOT/wordpress"
-	echo " * Starting project build"
-	sh .scripts/build.sh
-else
-	echo " * No build (.scripts/build.sh) script found."
+while true; do
+
+	read -p "Write your admin email: " ADMIN_EMAIL
+
+	if echo "${ADMIN_EMAIL}" | grep '^[a-zA-Z0-9]*@[a-zA-Z0-9]*\.[a-zA-Z]*$' >/dev/null; then
+		break
+	else
+		echo "Please write a valid email."
+	fi
+
+done
+
+if [ ! -d "wordpress" ]; then
+  mkdir "wordpress"
+  chmod -R a+w "wordpress"
 fi
 
+read -p "Wordpress repository, empty to clean wordpress: " REPOSITORY
+
 ## WORDPRESS SETUP ##
-if [ -f "./wordpress/wp-config.php" ]; then
-	echo "WordPress config file found."
-else
-	echo "WordPress config file not found. Installing..."
+if [ ! -z "${REPOSITORY}" ]; then
+	
+	echo "Local repository found. Downloading..."
+	git clone $REPOSITORY "$ROOT/wordpress"
+fi
+
+if [ ! -f "$ROOT/wordpress/wp-config.php" ]; then
```

Comment:
What if I have a wp-config.php on my repo, but not a WP installation (this is my current project use case)?